packages/salesforcedx-vscode-core/test/jest/commands/dataQuery.test.ts

```diff
@@ -160,42 +160,48 @@ describe('DataQuery Pure Functions', () => {
       expect(convertToCSV(records)).toBe(expected);
     });
 
-    it('should handle null and undefined values', () => {
-      const records = [{ Id: '001', Name: null, Description: undefined }];
-      const expected = 'Id,Name,Description\n001,,';
-      expect(convertToCSV(records)).toBe(expected);
-    });
+    describe('bad data handling', () => {
+      it('should handle null and undefined values', () => {
+        const records = [{ Id: '001', Name: null, Description: undefined }];
+        const expected = 'Id,Name,Description\n001,,';
+        expect(convertToCSV(records)).toBe(expected);
+      });
 
-    it('should handle object values', () => {
-      const records = [{ Id: '001', Owner: { Id: '005', Name: 'John' } }];
-      const expected = 'Id,Owner\n001,"{""Id"":""005"",""Name"":""John""}"';
-      expect(convertToCSV(records)).toBe(expected);
-    });
+      it('should handle object values', () => {
+        const records = [{ Id: '001', Owner: { Id: '005', Name: 'John' } }];
+        const expected = 'Id,Owner\n001,"{""Id"":""005"",""Name"":""John""}"';
+        expect(convertToCSV(records)).toBe(expected);
+      });
 
-    it('should handle malformed records with undefined first record', () => {
-      const records = [undefined, { Id: '001', Name: 'Test' }];
-      expect(convertToCSV(records)).toBe('');
-    });
+      it('should handle malformed records with undefined first record', () => {
+        const records = [undefined, { Id: '001', Name: 'Test' }];
+        // @ts-expect-error - bad input
```

Comment:
for when you know you're doing something the compiler won't allow, but maybe you want to test your very defensive handling